pkg/process/metadata/parser/java/jee.go

```diff
@@ -237,13 +249,15 @@ func ExtractServiceNamesForJEEServer(args []string, cwd string, fs afero.Fs) []s
 			value, ok := vendorWarFinder(fsCloser.fs)
 			_ = fsCloser.Close()
```

Comment:
very easy to have a breaking change the will cause FD leak
i do suggest to move that to a method and use defer

Reply:
Done thanks